modules/ROOT/pages/syntax-quick-reference.adoc

```diff
@@ -0,0 +1,1290 @@
+----
+
+You can control the video settings using xref:macros:audio-and-video.adoc[additional attributes and options] on the macro.
+
+== Source Code
+
+.Inline as monospace only
+[#ex-inline-code]
+----
+include::text:example$text.adoc[tag=b-mono-code]
+----
+
+.View result of <<ex-inline-code>>
+[%collapsible.result]
+====
+include::text:example$text.adoc[tag=b-mono-code]
+====
+
+.Inline as monospace and literal (i.e., literal monospace)
+[#ex-inline-literal]
+----
+include::pass:example$pass.adoc[tag=backtick-plus]
+----
+
+.View result of <<ex-inline-literal>>
+[%collapsible.result]
+====
+include::pass:example$pass.adoc[tag=backtick-plus]
+====
+
+.Literal and monospace line
+[#ex-literal-line]
+----
+include::verbatim:example$literal.adoc[tag=b-imp-code]
+----
+
+.View result of <<ex-literal-line>>
+[%collapsible.result]
+====
+include::verbatim:example$literal.adoc[tag=b-imp-code]
+====
+
+.Literal block
+[#ex-literal-block]
+----
+include::verbatim:example$literal.adoc[tag=b-block]
+----
+
+.View result of <<ex-literal-block>>
+[%collapsible.result]
+====
+include::verbatim:example$literal.adoc[tag=b-block]
+====
+
+.Listing block with title
+[#ex-listing]
+------
+include::verbatim:example$listing.adoc[tag=qr-listing]
+------
+
+.View result of <<ex-listing>>
+[%collapsible.result]
+====
+[caption="Listing 1. "]
+include::verbatim:example$listing.adoc[tag=qr-listing]
+====
+
+.Source block with title and syntax highlighting
+[#ex-highlight]
+------
+.Some Ruby code
+include::verbatim:example$source.adoc[tag=src-base]
+------
+
+.View result of <<ex-highlight>>
+[%collapsible.result]
+====
+[caption="Listing 1. "]
+.Some Ruby code
+include::verbatim:example$source.adoc[tag=src-base]
+====
+
+.Enable source highlighting
+[IMPORTANT]
+====
+xref:verbatim:source-highlighter.adoc[Source highlighting] is enabled by setting the `source-highlighter` attribute in the document header or passed as an argument.
+
+[source]
+----
+:source-highlighter: rouge
+----
+
+The built-in values are `coderay`, `highlightjs`, `rouge`, and `pygments`.
+====
+
+.Source block with callouts
+[#ex-callouts,subs=-callouts]
+------
+include::verbatim:example$callout.adoc[tag=b-src]
+------
+
+.View result of <<ex-callouts>>
+[%collapsible.result]
+====
+include::verbatim:example$callout.adoc[tag=b-src]
+====
+
+.Make callouts non-selectable
+[#ex-hide-callouts,subs=-callouts]
+------
+include::verbatim:example$callout.adoc[tag=b-nonselect]
+------
+
+.View result of <<ex-hide-callouts>>
+[%collapsible.result]
+====
+include::verbatim:example$callout.adoc[tag=b-nonselect]
+====
+
+.Source block content included from a file
+------
+include::verbatim:example$source.adoc[tag=src-inc]
+------
+
+.Source block content included from file relative to source directory
+------
+include::verbatim:example$source.adoc[tag=rel]
+------
+
+.Strip leading indentation from included file content
+------
+include::verbatim:example$source.adoc[tag=ind]
```

Comment:
nitpick: Maybe we shouldn't use the `lines` attributes in `include::lib/document.rb[lines=5..10]`. It might be confusing, does I need to add the `lines` attribute to strip leading indentation?

Alternatively, we could mention both `lines` and `tags` with a simple example and link to the dedicated section in the documentation to find out more?

Reply:
:+1:

Reply:
I added a lines and tags example to the "Include files" section and clarified why we combine indent with a partial include here.